TestApp/app/src/main/java/com/example/testapp/api/MovieService.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.testapp.api
+
+class MovieService {
```

Comment:
We should follow a Repository pattern. So Please rename this to MovieRepo

Reply:
implemented as suggested